radio/src/storage/yaml/yaml_datastructs_t12.cpp

```diff
@@ -236,6 +236,12 @@ const struct YamlIdStr enum_PotsWarnMode[] = {
   {  POTS_WARN_AUTO, "WARN_AUTO"  },
   {  0, NULL  }
 };
+const struct YamlIdStr enum_ModelOverridableEnable[] = {
+  {  OVERRIDE_VALUE_GLOBAL, "GLOBAL"  },
```

Comment:
@stephenaa did you patch this by hand? Normally, the code generator cuts only the first part, so that the correct generated value would be `VALUE_GLOBAL`, rather than `GLOBAL`. If we want to have it fixed without changing the YAML, you will need to patch the `enum` definition to look like this:
```
enum ModelOverridableEnable {
  OVERRIDE_VALUE_GLOBAL,
  OVERRIDE_VALUE_OFF,
  OVERRIDE_VALUE_ON
};
```

Reply:
Sorry, `enum` has been edited to better reflect what is needed.

Reply:
Got it :-)